frontend/src/screens/apps/AppCreated.tsx

```diff
@@ -192,8 +192,13 @@ export function ConnectAppCard({
             <p>App connected</p>
           </Badge>
         )}
-        <a href={pairingUri} target="_blank" className="relative">
-          <div className={!isQRCodeVisible ? "blur-md" : ""}>
+        <div className="relative">
+          <div
+            className={`${!isQRCodeVisible ? "blur-md cursor-pointer" : ""}`}
```

Comment:
You can use the `cn` util and just do `!isQRCodeVisible &&` instead of a ternary operator (see elsewhere)